### PR TITLE
Fix some typos & inconsistencies in Licensees

### PR DESCRIPTION
**Pull request by IsaacMarovitz**
Companies codes I cannot confirm:
- `San-X`, old code `0x25` new code "25"
- `.Entertainment i` old code `0x3C`
- `Yonezawa/s'pal`, old code `0x96` new code "96"

### CODE REVIEW

File: src/The_Cartridge_Header.md [hunk 1]
```diff
@@ -346,8 +346,8 @@ AA    | Broderbund
 AC    | Toei Animation    
 AD    | Toho
 AF    | Namco             
-B0    | acclaim           
-B1    | ASCII or Nexsoft
+B0    | Acclaim           
+B1    | ASCII Corportation or Nexsoft
```
Comment (ISSOtm) [inline src/The_Cartridge_Header.md:350]:
```suggestion
B1    | ASCII Corporation or Nexsoft
```

**Review by ISSOtm — CHANGES_REQUESTED** [review]
Thank you! That's a good quality improvement.

Do you have a source for those? It would be nice to be able to have some references, and (Wikipedia?) links would be a plus, even.

**Comment by IsaacMarovitz** [comment]
> Do you have a source for those? It would be nice to be able to have some references, and (Wikipedia?) links would be a plus, even.

As far as I can tell, this information was initially sourced from SephirothBCH's contribution on the gbdev wiki all the way back in 2000. I'm not sure where they got the original list from, and that list shares many of the issues present here.

**Comment by IsaacMarovitz** [comment]
For companies that underwent renames, should those be listed somewhere here?

**Comment by ISSOtm** [comment]
> For companies that underwent renames, should those be listed somewhere here?

Preferably. I'm thinking of using footnotes for that:

```md
- `xy`: [FooBar Interactive](//en.wikipedia.org/wiki/FooBar_Interactive)[^foobar]

<!-- And at the bottom of the page... -->

[^foobar]: Known as [Frobble Limited](//en.wikipedia.org/wiki/Frobble_Limited) since 2013.

```

> I can also provide links to each company's Wikipedia page (if they exist) if that's easier.

IMO, having both would be ideal.

**Review by avivace — APPROVED** [review]

**Review by ISSOtm — APPROVED** [review]
Outstanding work, thank you very much!!

**Pull request merged**